crates/bevy_render/src/camera/camera.rs

```diff
@@ -14,16 +14,65 @@ use bevy_transform::components::GlobalTransform;
 use bevy_window::{WindowCreated, WindowId, WindowResized, Windows};
 use serde::{Deserialize, Serialize};
 
+/// Camera component defining what to render.
+///
+/// The camera component is added to a world to define the properties of the viewpoint from
+/// which rendering occurs. It defines the position of the view to render, the projection method
+/// to transform the 3D objects into a 2D image, as well as the window into which that image
+/// is drawn.
+///
+/// Adding a camera is typically done by adding a bundle, either the [`OrthographicCameraBundle`]
+/// or the [`PerspectiveCameraBundle`].
+///
+/// The internal Bevy renderer makes a distinction between 2D and 3D content, which are rendered
+/// separately. To that end, at most two cameras should be added to the world, one per content.
```

Comment:
What about `UiCamera`?

Reply:
Honestly, no idea. I did not look into it at all.

Reply:
> at most two cameras should be added to the world, one per content.

IMO I think we should just cut this from the docs. It's not likely to be true for long (as we work on viewports), and UI cameras or other extensions break this assumption.

Reply:
Deprecated; `UiCamera` doesn't exist anymore.